src/schema/v2/homeView/zones/legacy.ts

```diff
@@ -0,0 +1,64 @@
+import { ResolverContext } from "types/graphql"
+import {
+  Auctions,
+  AuctionLotsForYou,
+  CuratorsPicksEmerging,
+  FeaturedFairs,
+  HeroUnits,
+  HomeViewSection,
+  LatestActivity,
+  LatestArticles,
+  LatestAuctionResults,
+  MarketingCollections,
+  NewWorksForYou,
+  NewWorksFromGalleriesYouFollow,
+  News,
+  RecentlyViewedArtworks,
+  RecommendedArtists,
+  ShowsForYou,
+  SimilarToRecentlyViewedArtworks,
+  TrendingArtists,
+  ViewingRooms,
+} from "schema/v2/homeView/sections"
+
+const LEGACY_ZONE_SECTIONS: HomeViewSection[] = [
+  LatestActivity,
+  NewWorksForYou,
+  HeroUnits,
+  // TK: active bids
+  AuctionLotsForYou,
+  Auctions,
+  LatestAuctionResults,
+  // TK: galleries near you
+  LatestArticles,
+  News,
+  CuratorsPicksEmerging,
+  MarketingCollections,
+  // TK: artwork recs
+  NewWorksFromGalleriesYouFollow,
+  RecommendedArtists,
+  TrendingArtists,
+  RecentlyViewedArtworks,
+  SimilarToRecentlyViewedArtworks,
+  ViewingRooms,
+  ShowsForYou,
+  FeaturedFairs,
+]
+
+export async function getLegacyZoneSections(context: ResolverContext) {
```

Comment:
Right now the zone logic boils down to this single function. 

I debated whether to introduce more structure here (e.g. a `Zone` base class with a defined interface, that individual zones such as `LegacyZone` could inherit from) but that all feels a bit premature right now, in the absence of clearer business rules for the zone logic.